### DISCUSSION TITLE
Should Eav datetime be stored as UTC instead?

### DISCUSSION THREAD

**Discussion by Hanmac** [OpenMage/magento-lts#2345] [Questions & Answers] [answered]
I don't know if it is a bug or a feature, and what would be the consequences.

i have a Product with Datetime eav attributes (stored in the extra datetime table)

now i want inside a block, show the date and the time of the attribute separate, i use the formatDate and formatTime functions
these does pass them to the core helper, but the helper thinks the values i passed to them should be utc time and transforms them

what would be the best way to stop the block from changing it?

**Comment by kiatng** [comment, marked as answer]
See PR #1525, it added a `$useTimezone` param in the function `formatDate() `.
